webapp/components/Comment/Comment.vue

```diff
@@ -54,6 +54,15 @@
         </button>
       </div>
       <ds-space margin-bottom="small" />
+      <base-button
+        :title="this.$t('post.comment.reply')"
+        icon="level-down"
+        class="reply-button"
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-01-22T15:02:28Z" title="Wednesday, January 22nd 2020, 4:02:28 pm +01:00">Jan 22, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -54,6 +54,15 @@
         </button>
       </div>
       <ds-space margin-bottom="small" />
+      <base-button
+        :title="this.$t('post.comment.reply')"
+        icon="level-down"
+        @click.prevent="reply"
```


Please put the events to the end of the list. 👇🏼

Reply:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-01-22T15:31:47Z" title="Wednesday, January 22nd 2020, 4:31:47 pm +01:00">Jan 22, 2020</time>_

---


@mattwr18 it's true, we haven't been following this guideline very strictly and I'm not sure that having things sorted alphabetically would do us much good – like it's handled in many other projects – but I'm with @Tirokk when he says "please put events at the end of the list" and would add "please put v-if and v-show and v-for in first position". Those are the two things I usually "fix" when I come across a different order...

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-01-23T05:58:31Z" title="Thursday, January 23rd 2020, 6:58:31 am +01:00">Jan 23, 2020</time>_

---


That’s why I sad:
> Please put the class *more to the beginning* of the list, see other examples.

Thanks for clarification @alina-beck ! I should have explicitly said that `v-if`, `v-else`, `v-else-if`, `v-show`, and `v-for` should be always at the beginning.

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-27T17:22:53Z" title="Monday, January 27th 2020, 6:22:53 pm +01:00">Jan 27, 2020</time>_

---


sorry, my comments were in `Discord`, so @alina-beck was responding to my comments there. 
I like the idea of having the events at the bottom, and the conditionals at the top, for sure.
I just didn't really like to ask @ogerly to change the location of his `class`es. The reasons, as I stated, are
```
1)  we haven't agreed as a team that is what we want to do and 
2) sometimes, I feel like we can ask @ogerly to change things, we don't ask others to do, 
```

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-27T17:22:53Z" title="Monday, January 27th 2020, 6:22:53 pm +01:00">Jan 27, 2020</time>_

---


sorry, my comments were in `Discord`, so @alina-beck was responding to my comments there. 
I like the idea of having the events at the bottom, and the conditionals at the top, for sure.
I just didn't really like to ask @ogerly to change the location of his `class`es. The reasons, as I stated, are
```
1)  we haven't agreed as a team that is what we want to do and 
2) sometimes, I feel like we can ask @ogerly to change things, we don't ask others to do, 
```